templates/tibiacom/index.php

```diff
@@ -843,6 +843,31 @@ class="carousel-control-next-icon"
     <br>
 </div>
 
+            if (MediumButtonOverDownload) {
+                MediumButtonOverDownload.style.visibility = 'hidden';
+                // Restante do código...
+            }
+        }
+    </script>
```

Comment:
preciso iniciar a tag <script> no inicio, e fechar só no fim de todas as funções.
mas onde está a função ?

```suggestion
<script>
   //         if (MediumButtonOverDownload) {
   //             MediumButtonOverDownload.style.visibility = 'hidden';
   //             // Restante do código...
   //         }
   //     }
```